main/glossary/README.md

```diff
@@ -159,7 +159,7 @@ If they have ten houses for sale, they have ten different contract instances.
 
 ## Creator Invitation
 
-An [invitation](#invitation) optionally returned by [`startInstance()`](/reference/zoe-api/zoe.md#e-zoe-startinstance-installation-issuerkeywordrecord-terms-privateargs) that the contract instance
+An [invitation](#invitation) optionally returned by [`E(zoe).startInstance(...)`](/reference/zoe-api/zoe.md#e-zoe-startinstance-installation-issuerkeywordrecord-terms-privateargs) that the contract instance
```

Comment:
It's a hope, with no concrete plan behind it. Not worth covering in the docs at this point.

Reply:
It might also be worth noting that `startInstance` is subsumed into `startUpgradable`. (It is, right?)

Reply:
> It might also be worth noting that startInstance is subsumed into startUpgradable. (It is, right?)

That doesn't sound familiar. I think you're misremembering something, though I don't know what.

Reply:
I dug into this. The Agoric chain home object can provide `startUpgradable` as a thin wrapper for `E(zoe).startInstance`. It’s provided by `vats/src/core/basic-behaviors.js`. This is why we were unable to find `startInstance` in the core proposal in the `dapp-game-places` template during the hackathon. We eventually found `startUpgradable` and from there were able to figure out how to thread `offerArgs` thru the level of indirection.

Reply:
`startUpgradable` is a power from the bootstrap environment; it's available to core-evals, but it's not part of the Zoe API.

The purpose of `startUpgradable` is somewhat relevant, though: it's to make sure we don't lose admin facets needed for upgrade.

Do we tell folks in the Zoe docs that startInstance() also returns an admin facet, and if you lose it, you will NEVER BE ABLE TO UPGRADE?

Reply:
`startUpgradable` should be covered in [docs for bootstrap powers](https://docs.agoric.com/guides/coreeval/permissions.html#declaring-required-capabilities). Those are pretty thin. That's probably worth a separate issue...

Reply:
I looked for a relevant issue and found:
 - https://github.com/Agoric/agoric-sdk/issues/5775

But it's one of those "do X" issues that doesn't say _why_ do X... other than
> That functionality can be supplied from the creatorFacet.

Reply:
Thanks for finding that. In light of @samsiegart 's two comments at https://github.com/Agoric/agoric-sdk/issues/5775#issuecomment-1561870319 , I withdraw the request to emphasize the deprecation of creator invitation until Sam's question is resolved.